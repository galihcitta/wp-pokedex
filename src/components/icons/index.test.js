import React from "react";
import Icons from "./index";

describe("Icons Component", () => {
    it("renders and create snapshot <Icons />", () => {
        expect(toJson(<Icons name="burgerMenu" />)).toMatchSnapshot();
    });

    it("check width in style components <Filter />", () => {
        const tree = renderer.create(<Icons name="logo" />).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree).toHaveStyleRule("background-image", "url(ic-logo.svg)");
    });
});
