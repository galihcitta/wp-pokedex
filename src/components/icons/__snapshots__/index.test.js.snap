// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons Component check width in style components <Filter /> 1`] = `
.c0 {
  background-color: transparent;
  background-image: url(ic-logo.svg);
  background-position: center center;
  background-position: 50%;
  background-repeat: no-repeat;
  background-size: 18px 16px;
  background-size: 30px 30px;
  display: block;
  font-size: 0;
  height: 30px;
  margin: auto;
  text-indent: -99999px;
  width: 30px;
}

<i
  className="c0"
  name="logo"
/>
`;

exports[`Icons Component renders and create snapshot <Icons /> 1`] = `undefined`;
